docs/source/conf.py

```diff
@@ -166,8 +177,26 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# TEMPORARILY DISABLED
-#html_static_path = ['_static']
+html_theme = 'pydata_sphinx_theme'
+html_theme_options = {
+    'external_links': [
+        {'url': 'http://www.aiida.net/', 'name': 'AiiDA Home'}
+    ],
+    'github_url': 'https://github.com/aiidateam/aiida-core',
+    'use_edit_page_button': True,
+}
+html_context = {
+    'github_user': 'aiidateam',
+    'github_repo': 'aiida-core',
+    'github_version': 'master',
+    'doc_path': 'docs',
```

Comment:
```suggestion
    'doc_path': 'docs/source',
```

Reply:
So that "edit this page" links to the right place

Reply:
Thank you!